index.html

```diff
@@ -0,0 +1,132 @@
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button id="btnSee">See Project</button>
+        </div>
+
+
+    </section>
+    <section class="about">
+        <div class="about-items">
+            <h2>About me</h2>
+        </div>
+        <div class="about-items">
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="about-items">
+            <button>
+                Get My Resume
+            </button>
+                
+        </div>
+        <div class="about-items">
+            <img class="background" src="./img/Group.png" alt="">
+        </div>
+        <div class="about-items">
+            <hr>
+        </div>
+        <div class="about-items">
+            <ul>
+                <li>Languages
+                    <ul class="sublist">
+                        <li>Javascript</li>
```

Comment:
- [ ] Delete bullet points from technologies lists.
![Captura de pantalla_2021-09-18_02-37-34](https://user-images.githubusercontent.com/53111464/133880717-cedb4b30-50d5-424c-a2f2-7b18c362f88c.png)

- [ ] For every list use `font-size, font-family, font-weight` values same as Figma template.

Reply:
I wish you had looked at the CSS codes and you would have seen the font family and font-size specification
i deleted now font-family